torch/fx/_pytree.py

```diff
@@ -1,32 +1,37 @@
 from collections import namedtuple
-from typing import Any, Callable, Dict, List, NamedTuple, Tuple, Type, Optional
+from typing import Any, Callable, Dict, List, NamedTuple, Optional, Tuple, Type
```

Comment:
While you are doing this, should this file be removed from UFMT excludes in lintrunner?

Reply:
Updated.